test/fake_3par_data.py

```diff
@@ -0,0 +1,376 @@
+                    'flash_cache': None,
+                    'snapshots': []}
+
+volume_dedup_compression = {'name': VOLUME_NAME,
+                            'id': VOLUME_ID,
+                            'display_name': 'Foo Volume',
+                            'size': 16,
+                            'host': FAKE_CINDER_HOST,
+                            'volume_type': 'dedup_compression',
+                            'volume_type_id': VOL_TYPE_ID_DEDUP_COMPRESS}
+
+volume_dedup = {'name': VOLUME_NAME,
+                'id': VOLUME_ID,
+                'display_name': 'Foo Volume',
+                'size': 2,
+                'host': FAKE_CINDER_HOST,
+                'volume_type': 'dedup',
+                'volume_type_id': VOLUME_TYPE_ID_DEDUP,
+                'provisioning': DEDUP,
+                'flash_cache': None,
+                'snapshots': []}
+
+volume_qos = {'name': VOLUME_NAME,
+              'id': VOLUME_ID,
+              'display_name': 'Foo Volume',
+              'size': 2,
+              'host': FAKE_CINDER_HOST,
+              'volume_type': None,
+              'volume_type_id': 'gold'}
+
+volume_flash_cache = {'name': VOLUME_NAME,
+                      'id': VOLUME_ID,
+                      'display_name': 'Foo Volume',
+                      'size': 2,
+                      'host': FAKE_CINDER_HOST,
+                      'volume_type': None,
+                      'volume_type_id': VOLUME_TYPE_ID_FLASH_CACHE,
+                      'provisioning': THIN,
+                      'flash_cache': 'true',
+                      'snapshots': []}
+
+wwn = ["123456789012345", "123456789054321"]
+
+connector = {'ip': '10.0.0.2',
+             'initiator': 'iqn.1993-08.org.debian:01:222',
+             'wwpns': [wwn[0], wwn[1]],
+             'wwnns': ["223456789012345", "223456789054321"],
+             'host': FAKE_HOST,
+             'multipath': False}
+
+connector_multipath_enabled = {'ip': '10.0.0.2',
+                               'initiator': ('iqn.1993-08.org'
+                                             '.debian:01:222'),
+                               'wwpns': [wwn[0], wwn[1]],
+                               'wwnns': ["223456789012345",
+                                         "223456789054321"],
+                               'host': FAKE_HOST,
+                               'multipath': True}
+
+volume_type = {'name': 'gold',
+               'deleted': False,
+               'updated_at': None,
+               'extra_specs': {'cpg': HPE3PAR_CPG2,
+                               'qos:maxIOPS': '1000',
+                               'qos:maxBWS': '50',
+                               'qos:minIOPS': '100',
+                               'qos:minBWS': '25',
+                               'qos:latency': '25',
+                               'qos:priority': 'low'},
+               'deleted_at': None,
+               'id': 'gold'}
+
+volume_type_replicated = {'name': 'replicated',
```

Comment:
Can this be removed so that we have only needed definitions and declarations

Reply:
Sure